gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/equality/EqualityTest.java

```diff
@@ -200,10 +208,11 @@ public void testUndirectedCollectionEquality() throws Exception {
   public void testGraphEqualityByElementIds() throws Exception {
     FlinkAsciiGraphLoader loader = getTestGraphLoader();
 
-    GraphEquality equality = new GraphEquality(
-      new GraphHeadToEmptyString(),
-      new VertexToIdString(),
-      new EdgeToIdString(),
+    GraphEquality<EPGMGraphHead, EPGMVertex, EPGMEdge, LogicalGraph, GraphCollection> equality =
+      new GraphEquality<>(
+      new GraphHeadToEmptyString<>(),
```

Comment:
Indentation: arguments to GraphEquality constructor should have more indentation.